### DISCUSSION TITLE
Redocusaurus for Docusaurus v3.1.1

### DISCUSSION THREAD

**Discussion by malathikrishnap** [rohit-gohri/redocusaurus#327] [Q&A] [answered]
Hi there,
What's the latest version of Redocusaurus? Is it fully compatible with Docusaurus v3.1.1?
If yes, could you please share the steps to integrate Redoc with Docusaurus 3.1.1?

**Comment by Nostromos** [comment]
@rohit-gohri Came upon this question and wanted more information before going further - 

1. Do you plan to update for Docusaurus v3+?
2. If so, do you have the time/motivation to do so? Do you want/need help? 

If you don't plan on updating for Docasaurus 3, I'll probably fork this and do what I can! If you'd like to make the update but have a life/work, I'll check out the contribution guidelines and see if I can be helpful.

**Reply by rohit-gohri** [reply]
Hey! I'm not actively using this in my own projects anymore, so I only work on this when I have free time (& motivation 😅). 

But I'm always happy to take contributions and do a release as most of it is automated so doesn't require a lot of effort

**Reply by rohit-gohri** [reply]
Redocusaurus should already work with 3, are you facing any issues? Please open issues if you are

https://github.com/rohit-gohri/redocusaurus/blob/main/website%2Fpackage.json#L25

**Comment by rohit-gohri** [comment, marked as answer]
Hey the current latest version is 2 and it was last tested with docusaurus v3.0.0 so it should work with 3.1 too. Please open issues if it doesn't 

See the example that powers the docs website:
https://github.com/rohit-gohri/redocusaurus/blob/main/website%2Fpackage.json#L25

**Reply by Nostromos** [reply]
Sorry! Still new to this and didn't think to check your package.json to see if it was already compatible.

I'll get this set up with my docs up at some point today or this week and let you know if anything's not working. If there are issues, and depending on their complexity, I'll try to give you a good PR. Maybe there are new Docasaurus 3 features that aren't covered (not sure)? 

Also, thanks so much for the quick response. I'll make sure to check your contribution guidelines and include a changeset if I propose any adds/fixes.

**Reply by rohit-gohri** [reply]
There is this issue that's about making use of a new feature in v3 - https://github.com/rohit-gohri/redocusaurus/issues/321

But I wouldn't try it unless you are already familiar with internals of [redoc](https://github.com/Redocly/redoc/) ( the package we use to parse and render the docs )

**Reply by Nostromos** [reply]
Maybe a good opportunity to get familiar with it! I'll dig in and see whether it's too much for me. Thanks!
